### PR TITLE
Repalce getpaper with arxiv API

### PR DESCRIPTION
**Pull request by takahirosir**
I'm going to replace the getpaper function with [arxiv API](https://pypi.org/project/arxiv/).

### CODE REVIEW

File: paper_class.py [hunk 1]
```diff
@@ -105,33 +108,32 @@ def get_papers(search_results,
...
-    print(f"Searching {query} from arxiv... (max #results: {max_results})")
+    # print(f"Searching {query} from arxiv... (max #results: {max_results})")
     # Perform the arXiv search using the arXiv API
     # base_url: str = "http://export.arxiv.org/api/query?"
     # search_query: str = f"search_query=all:{query}&start=0&max_results={max_results}&sortBy={sort_type}&sortOrder={sort_order}"
 
     # response: requests.Response = requests.get(base_url + search_query)
     # feed: feedparser.FeedParserDict = feedparser.parse(response.text)
 
+    # Create a directory to store the downloaded papers
```
Comment (Quartzing) [inline paper_class.py:119]: remove

File: paper_class.py [hunk 2]
```diff
@@ -151,11 +153,22 @@ def get_papers(search_results,
...
-        query="LLM medical",
-        max_results=10,
+        search,
         download=False,
     )
+    # results: Dict[str, Paper] = get_papers(
+    #     query="LLM medical",
+    #     max_results=10,
+    #     download=False,
+    # )
```
Comment (Quartzing) [inline paper_class.py:171]: remove

**Review by Quartzing — CHANGES_REQUESTED** [review]

File: paper_class.py [hunk 3]
```diff
@@ -151,11 +153,22 @@ def get_papers(search_results,
     # return results
```
Comment (Quartzing) [inline paper_class.py:153]: remove

File: paper_class.py (hunk 2 above)
Comment (takahirosir) [reply paper_class.py:171]: done

File: paper_class.py (hunk 3 above)
Comment (takahirosir) [reply paper_class.py:153]: done

File: paper_class.py (hunk 1 above)
Comment (takahirosir) [reply paper_class.py:119]: done

File: paper_class.py [hunk 4]
```diff
@@ -105,55 +108,34 @@ def get_papers(search_results,
...
-    print(f"Searching {query} from arxiv... (max #results: {max_results})")
-    # Perform the arXiv search using the arXiv API
-    # base_url: str = "http://export.arxiv.org/api/query?"
-    # search_query: str = f"search_query=all:{query}&start=0&max_results={max_results}&sortBy={sort_type}&sortOrder={sort_order}"
-
-    # response: requests.Response = requests.get(base_url + search_query)
-    # feed: feedparser.FeedParserDict = feedparser.parse(response.text)
+    output_directory: str = "arxiv_papers"
+    os.makedirs(output_directory, exist_ok=True)
+    print("test")
```
Comment (Quartzing) [inline paper_class.py:114]: are these useful?

**Review by Quartzing — CHANGES_REQUESTED** [review]

File: paper_class.py (hunk 4 above)
Comment (takahirosir) [reply paper_class.py:114]: done

**Review by Quartzing — APPROVED** [review]
thx!

**Pull request merged**